src/services/TaskRepository.ts

```diff
@@ -1,83 +1,69 @@
-import { QueryTypes } from "sequelize";
+import { v4 as uuidv4 } from 'uuid';
+const Task = require('../models/SequelizeModels');
 
-const instances = require('hapi-sequelizejs').instances;
 export class TaskRepo {
-    
-    async getTasks(completion:string, sortBy:string, sortOrder:string){
-        var queryString = "SELECT * FROM Tasks ";
 
-        if (completion != null && completion != undefined)
-        {
-            queryString += `WHERE completed = ${completion} `;
+    async getTasks(completion:string, sortBy:string, sortOrder:string){
```

Comment:
Complete and Incomplete filtering does not work.

Reply:
Fixed